src/main/java/jpos/profile/ProfileException.java

```diff
@@ -29,18 +29,28 @@ public class ProfileException extends Exception
 	// Ctor(s)
 	//
 
+	/**
+	 * 
```

Comment:
Meaningful JavaDoc comment missing

Reply:
I removed the empty comment. It had been added automatically by Eclipse when generating the serial UID to solve a compiler warning.